lib/sen_tweet_web/router.ex

```diff
@@ -13,14 +14,20 @@ defmodule SenTweetWeb.Router do
     plug :accepts, ["json"]
   end
 
+  pipeline :auth do
+    plug :basic_auth, username: "bitfeels", password: "password"
+  end
+
   scope "/", SenTweetWeb do
     pipe_through :browser
 
     get "/", PageController, :index
+    live "/live", PageLive, :index
   end
 
-  # Other scopes may use custom stacks.
-  # scope "/api", SenTweetWeb do
-  #   pipe_through :api
-  # end
+  scope "/admin", SenTweetWeb do
+    pipe_through [:browser, :auth]
```

Comment:
the `/admin` scope is piped through `:auth`, which seems to indicate that username="bitfeels" and password="password", but instead I need to copypasta the env variable you sent me. What am I missing?